### ISSUE TITLE
validate / support resolution of various common packages in node modules

### ISSUE THREAD

**Issue by thescientist13** [ProjectEvergreen/greenwood#429]
## Type of Change
- [x] New Feature Request
- [ ] Documentation / Website
- [ ] Improvement / Suggestion
- [ ] Bug
- [ ] Other (please clarify below)

## Summary
As part of #355 and #418 , with the refactor the resolution of _node_modules_ to import maps and Rollup, we need to make sure we handle a good variety of packages installable via npm / yarn.  Right now it's mostly just ESM based packages being validated (**lit-html** / **lit-element**).

## Details
Basically what [**Snowpack**](https://www.snowpack.dev/) is doing, so we should test things like
- moment
- lodash
- others?

We should make PRs to test these and fix.  I suspect most of the support will come from CJS based packages.  

Maybe we can / should just incorporate Snowpack? 🤔
